File: 151001/Manchik/publisher/src/main/java/application/dto/AuthorResponseTo.java
package application.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthorResponseTo implements Serializable {
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151004/Afanasenko/appLab4/common/src/main/kotlin/com/github/kornet_by/dc/lab4/dto/response/AuthorResponseTo.kt
package com.github.kornet_by.dc.lab4.dto.response

import kotlinx.serialization.Serializable

@Serializable
data class AuthorResponseTo(
	private val id: Long?,
	private val login: String,
	private val password: String,
	private val firstname: String,
	private val lastname: String
)